src/components/MoneyRequestConfirmationList.js

```diff
@@ -273,6 +278,36 @@ function MoneyRequestConfirmationList(props) {
         );
     }, [confirm, props.selectedParticipants, props.bankAccountRoute, props.iouCurrencyCode, props.iouType, props.isReadOnly, props.policyID, selectedParticipants, splitOrRequestOptions]);
 
+    /**
+     * Grab the appropriate image URI based on file type
+     *
+     * @param {String} receiptPath
+     * @param {String} receiptSource
+     * @returns {*}
+     */
+    const getImageURI = (receiptPath, receiptSource) => {
+        const {fileExtension} = FileUtils.splitExtensionFromFileName(receiptSource);
+        const isReceiptImage = Str.isImage(props.receiptSource);
```

Comment:
Heads up that this won't work with HEIC images

Reply:
When you say it doesn't work, do you mean that HEIC images will not preview on web and we will show the default fallback image? I think that's expected for now.

Reply:
Cool, just wanted to clarify 👍🏼 

I think converting HEIC to jpg for cross-platform display is a separate convo happening somewhere, using the fallback for now is fine